spec/css/style_rule.hrx

```diff
@@ -215,3 +167,217 @@ a {b: c //
 a {
   b: c;
 }
+
+<===>
+================================================================================
+<===> declaration/mixed/after_style_rule/higher_specificity/input.scss
+a {
+  b {c: d}
+  e: f;
+}
```

Comment:
IIUC this is OK because `a` will never be `b`, so their relative order doesn't matter, right?

It would have been different if these were `.a` and `.b`. Is there a test for this case already?

Reply:
Good question. This actually doesn't take into account whether the selectors can overlap; it's considered okay because `a b` has a different specificity than `a`, so it'll always resolve in specificity order rather than document order. But it would be better to use class selectors to make that explicit.

Reply:
oh, it's due to specificity!